### ISSUE TITLE
Support Texy syntax (through a web service)

### ISSUE THREAD

**Issue by OndraZizka** [jbake-org/jbake#474]
There is a great but not too famous markup language: [Texy](https://texy.info/en/).
It has more features than Markdown, and (subjectively) more user friendly.
But since the PHP implementation (not the language) it's quite complex, it didn't get fully compatible ports to JS, Java or Python, and didn't spread much beyond PHP world.

It's written in PHP, and I have ported it to Java, but only partly.
So I have created a Docker image of a web service. A POST request with the markup text returns the converted XHTML.

I suggest to make JBake support Texy syntax through that service.  
I could write a class that would handle calls to this service. It could be parallelized. What I need is to have some JBake callback interface, which would support paralelization and synchronous calls (I assume blocking calls is not a good idea for performance reasons).

My question is - would you (JBake community) support this? I.e. if done well, would that get merged?

**Comment by jonbullock** [comment]
My initial thought is that I'm not keen on depending on a 3rd party web service for support for a content format. My preference would be to use the port.

Eager to hear what he community thinks about this topic though...

**Comment by ancho** [comment]
I interpreted it as a selfhosted service which uses the php texy library. As long as there is enough flexibility to configure the engine I don't see a big problem. Nobody is forced to use it.

**Comment by OndraZizka** [comment]
Right, there would be an option, how about `converter.texy.enable=true|false`.
And when JBake turns to plugins, this could be a matter of including or not the .jar.

**Comment by jonbullock** [comment]
Ahh OK in that case then I'm good with it, need to make sure the situation is explained in the docs well.
